.github/workflows/helm_charts_scalar.yml

```diff
@@ -45,6 +45,43 @@ jobs:
       - name: Run chart-testing (lint)
         run: ct lint --config .github/ct.yaml
 
+  kubeaudit-chart:
+    runs-on: ubuntu-latest
+    env:
+      KUBE_AUDIT_VERSION: 0.16.0
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      - name: Set up promql
+        uses: supplypike/setup-bin@v1
+        with:
+          uri: "https://github.com/Shopify/kubeaudit/releases/download/${{ env.KUBE_AUDIT_VERSION }}/kubeaudit_${{ env.KUBE_AUDIT_VERSION }}_linux_amd64.tar.gz"
+          name: "kubeaudit"
+          version: ${{ env.KUBE_AUDIT_VERSION }}
+
+      - name: Add Helm repos
+        run: |
+          helm repo add scalar https://scalar-labs.github.io/helm-charts
+
+      - name: Run kubeaudit
+        run: |
+          # TODO If more charts are supported by kubeaudit, they should be added.
+          # Change to `ls charts` when all charts are supported.
+          chart_dirs=(envoy)
```

Comment:
First of all, only `envoy-charts` have been audited, so other `charts` will be added as soon as they are audited.